scripts/projectUpdater.ps1

```diff
@@ -325,6 +358,19 @@ _openMergeWindow `
     $projectFolder/.conf/tmp/.gitignore `
     $projectFolder/.gitignore
 
+# DEPS.JSON:
+_openMergeWindow `
+    $projectFolder/.conf/tmp/deps.json `
+    $projectFolder/.conf/deps.json
```

Comment:
Do we really want this? IMHO would be better and more secure to only accept updates on the `deps.json` and scripts without user interaction.

Reply:
This one is mainly because of the addition of the scripts. It is very possible that the use has some script that his project depends on and he will add it there. I see this being used on a project that has more than one developer for example. Once the developer shares the project with the other he wants it to install also some dependency or installation script that his project depends on.

Reply:
ok, make sense. Let's leave this way then